x-pack/test/plugin_api_integration/test_suites/task_manager/task_manager_integration.js

```diff
@@ -479,8 +495,13 @@ export default function ({ getService }) {
         expect(
           docs.filter((taskDoc) => taskDoc._source.taskId === longRunningTask.id).length
         ).to.eql(1);
+
+        const task = await currentTask(longRunningTask.id);
+        expect(task.status).to.eql('running');
       });
 
+      await delay(1000);
```

Comment:
Is there a reason for a delay after a `retry.try`? Seems like a future cause of flakiness. The logic of the code above could be changed to wait until the task finished running.

Reply:
oh, that was a mistake - this was meant to be `await ensureTasksIndexRefreshed();` too 👍 
Good catch!